sdk/areas/areas.go

```diff
@@ -96,12 +97,12 @@ func (c *Client) GetArea(ctx context.Context, userAuthToken, serviceAuthToken, c
 	if next, ok := body["next"]; ok && (serviceAuthToken != "" || userAuthToken != "") {
 		b, err = json.Marshal(next)
 		if err != nil {
-			return
+			return areaDetails, err
```

Comment:
If we are returning an error, we probably should not return a populated `AreaDetails` struct. Normally we'd return `nil, err` but that would require changing the function signiture to return a pointer (`(*AreaDetails, error)`). Although doing this would have the added benefit that the other `areaDetails, err` returns further above in the function could also be changed to `nil, err` too which would be preferable.

Reply:
Yes, completely agree. The reason why I have not done it is this was just a pr to enable to linting and fix the lint issues that is raised by the linter. So I wasn't keen on refactoring anything unless otherwise raised by the linter.

Reply:
But as we're changing this line in this PR I'd fix the underlining issue here too. If you can't change the function signature I'd explicitly set an empty struct here instead. Eg.
```suggestion
			return AreaDetails{}, err
```